package.json

```diff
@@ -27,7 +27,7 @@
     "electron-store": "^6.0.0",
     "electron-updater": "^4.2.0",
     "eventemitter3": "^4.0.7",
-    "font-scanner": "dkdkwizard/font-scanner",
+    "font-scanner": "^0.1.2",
```

Comment:
The font scanner has some issue, so I fork it and modify some places.

Reply:
updated